src/WingetCreateCLI/Common.cs

```diff
@@ -61,6 +64,39 @@ public static void CleanUpFilesOlderThan(string cleanUpDirectory, int cleanUpDay
             }
         }
 
+        /// <summary>
+        /// Gets the path for display. This will anonymize the path if caller provides the appropriate flag.
+        /// </summary>
+        /// <param name="path">Path to be displayed.</param>
+        /// <param name="substituteEnvironmentVariables">Whether or not to substitute environment variables.</param>
+        /// <returns>Anonymized path or original path.</returns>
+        public static string GetPathForDisplay(string path, bool substituteEnvironmentVariables = true)
```

Comment:
We should add a unit test to verify the behavior of this.

Reply:
Unit test added under `CommonTests.cs` file